docs/DeveloperGuide.md

```diff
@@ -730,13 +798,45 @@ Similar to [<ins>UC22](#use-case-uc22-assign-a-contact-to-one-or-more-weddings)
     * 2a1. System shows an error message prompting the user to put in a valid index.
 
       Use case resumes at step 1.
+  
+* 2b. The contact is already an assigned Vendor.
+    * 2a1. System displays an error message.
+
+      Use case resumes at step 1.
 
 
 ### **Use case: UC25 — Unassigning a Vendor**
-Similar to [<ins>UC24](#use-case-uc24assigning-a-vendor) except un-assigning a vendor and an additional extension.
+
+**MSS**
+
+1. User <ins>lists all contacts [(UC01)](#use-case-uc01list-all-contacts)</ins>.
+2. User requests to unassign the Vendor designation from a contact.
```

Comment:
Bit technical, can change to: "User requests to unassign the contact as a vendor."

Reply:
will do